pkg/errors/errors.go

```diff
@@ -210,6 +210,9 @@ var (
 
 	// ErrTimeout is raised when an operation has timed out
 	ErrTimeout = fmt.Errorf("operation timed out")
+
+	//ErrDevModeNotValid is raised when development mode in manifest is not 'sync' nor 'hybrid'
```

Comment:
I want to open a discussion here about if the errors should be defined on the package that is producing the error or in a common/utils file. I think it's better to have it in the serializer file but I'm nitpicking

Reply:
+1, if is not necessary out of the package let's keep it inside. I will update it

Reply:
about the error, should this be a `UserError` instead?

Reply:
sounds good, let's keep the change simple

Reply:
refactor done d21bcfa484a558045b528d7db392286dd7bd9b42